### ISSUE TITLE
Saved model without pretraining

### ISSUE THREAD

**Issue by hojunson** [ydhongHIT/DDRNet#26]
Hello Yuanduo,

I think your achievement is really awesome. I am looking into your code and your model to develop more on it.

I was wondering whether the "best.pth" model is the trained model without Imagenet pre-training. 

I remember I got the file from the google drive and it shows about 75.xx in validation set of Cityscapes. 

Could you confirm it?

**Comment by ydhongHIT** [comment]
> Hello Yuanduo,
> 
> I think your achievement is really awesome. I am looking into your code and your model to develop more on it.
> 
> I was wondering whether the "best.pth" model is the trained model without Imagenet pre-training.
> 
> I remember I got the file from the google drive and it shows about 75.xx in validation set of Cityscapes.
> 
> Could you confirm it?

All the models utilize the imagenet pre-training. 75.xx is abnormal. There are some third-party repos or papers reproducing my method and it is easy to achieve above 77% mIoU for ddrnet_23_slim. The model I provided should be 77.8.

**Comment by ydhongHIT** [comment]
> Hello Yuanduo,
> 
> I think your achievement is really awesome. I am looking into your code and your model to develop more on it.
> 
> I was wondering whether the "best.pth" model is the trained model without Imagenet pre-training.
> 
> I remember I got the file from the google drive and it shows about 75.xx in validation set of Cityscapes.
> 
> Could you confirm it?

If you have some problems, you can refer to this [repo](https://github.com/chenjun2hao/DDRNet.pytorch).
